mobile/Screen/EventScreen.js

```diff
@@ -23,109 +25,103 @@ const styles = StyleSheet.create({
+    exhibit: require('../assets/eventThumbnails/exhibit.png'),
+    film: require('../assets/eventThumbnails/film.png'),
+    music: require('../assets/eventThumbnails/music.png'),
+    performance: require('../assets/eventThumbnails/performance.png'),
+    'spoken word': require('../assets/eventThumbnails/spokenWord.png'),
+    miscellaneous: require('../assets/eventThumbnails/miscellaneous.png'),
+    'visual art': require('../assets/eventThumbnails/visualArt.png'),
+  };
+
+  const getAllEvents = async () => {
+    try {
+      const result = await axios.get(`${URL}/event/get`);
+      setAllEvents(result.data);
+      setFilteredEvents(result.data);
+      return result.data;
+    } catch (err) {
+      console.error(err);
+      return err;
+    }
+  };
+
+  useEffect(() => {
+    getAllEvents();
+  }, []);
 
   const handleButtonPress = () => {
     setShowCalendar(!showCalendar);
   };
 
+  const getWeek = (date) => {
+    const firstDayOfMonth = new Date(date.getFullYear(), date.getMonth(), 1);
+    const dayOfWeek = firstDayOfMonth.getDay() || 7; // Convert Sunday to 7
+    const dayOfMonth = date.getDate();
+
+    const adjustedDayOfMonth = dayOfMonth + dayOfWeek - 1;
+    const weekOfMonth = Math.ceil(adjustedDayOfMonth / 7);
+
+    return weekOfMonth;
+  };
+  // const weekNum = (date, weekday) => {
+  //   const firstDay = new Date(date.getFullYear(), date.getMonth(), 1);
+
+  //   let count = 0;
+  //   for (let day = firstDay; day <= date; day.setDate(day.getDate() + 1)) {
+  //     if (day.getDay() === weekday) {
+  //       count += 1;
+  //     }
+  //   }
+  //   console.log(count);
+  //   return count;
+  // };
+
+  // the way the calendar works is you have to double click it to select another date
+  const filterEvents = (date) => {
+    if (!date) {
+      setFilteredEvents(allEvents);
+    } else {
+      const inputDate = new Date(date);
+      inputDate.setHours(17, 0, 0);
+      const filtered = allEvents.filter((event) => {
+        const eventDate = new Date(event.EventData.dateDetails.date);
+        const dayOfWeek = event.EventData.dateDetails.day;
+        const rec = event.EventData.dateDetails.recurring;
+        const weekOfMonth = event.EventData.dateDetails.week;
+
+        if (rec === 'Weekly') {
+          return (inputDate >= eventDate && inputDate.getDay() === dayOfWeek);
+        }
+        if (rec === 'Monthly') {
+          return (inputDate >= eventDate && inputDate.getDay() === dayOfWeek && getWeek(inputDate) === weekOfMonth)
+        }
+        return (inputDate.getTime() === eventDate.getTime());
+      });
+      setFilteredEvents(filtered);
+    }
+  };
+
   const handleDateSelect = (date) => {
     setSelectedDate(date);
     setShowCalendar(false);
+    filterEvents(date);
   };
 
-  const filteredEvents = events.filter(
-    (event) => !selectedDate || new Date(event.date) >= new Date(selectedDate),
-  );
-
   return (
     <ScrollView style={styles.scrollView}>
```

Comment:
Part of the hi-fis includes the header ("Events") being formatted with the font and everything else.  There's an option to hide the header on Screens called `header shown`, which you can set to `false` in the Stack Navigator in `App.js` (which, when you merge main back into this branch, you'll see is now in a component called `AppNavigator.js`) (as in https://github.com/lablueprint/skid-row-arts-alliance/blob/art-details-screen/mobile/Components/AppNavigator.js#L187); this will delete the entire header and allow you to create a custom one like in the hi-fi/final design